docs/dev/python/user-guide/table/10_minutes_to_table_api.md

```diff
@@ -0,0 +1,712 @@
+
+# 2. create source Table
+table_env.execute_sql("""
+CREATE TABLE datagen (
+ id INT,
+ data STRING
+) WITH (
+ 'connector' = 'datagen',
+ 'fields.id.kind' = 'sequence',
+ 'fields.id.start' = '1',
+ 'fields.id.end' = '10'
+)
+""")
+
+# 3. create sink Table
+table_env.execute_sql("""
+CREATE TABLE print (
+ id INT,
+ data STRING
+) WITH (
+ 'connector' = 'print'
+)
+""")
+
+# 4. query from source table and caculate
+# create a Table from a Table API query:
+tapi_result = table_env.from_path("datagen").select("id + 1, data")
+# or create a Table from a SQL query:
+sql_result = table_env.sql_query("SELECT * FROM datagen").select("id + 1, data")
+
+# 5. emit query result to sink table
+# emit a Table API result Table to a sink table:
+tapi_result.execute_insert("print").get_job_client().get_job_execution_result().result()
+sql_result.execute_insert("print").get_job_client().get_job_execution_result().result()
+# or emit results via SQL query:
+table_env.execute_sql("INSERT INTO print SELECT * FROM datagen").get_job_client().get_job_execution_result().result()
+
+{% endhighlight %}
+
+{% top %}
+
+Create a TableEnvironment
+-------------------------
+
+The `TableEnvironment` is a central concept of the Table API and SQL integration. The following code example shows how to create a TableEnvironment:
+
+{% highlight python %}
+
+from pyflink.table import EnvironmentSettings, StreamTableEnvironment, BatchTableEnvironment
+
+# create a blink streaming TableEnvironment
+table_env = StreamTableEnvironment.create(environment_settings=EnvironmentSettings.new_instance().in_streaming_mode().use_blink_planner().build())
+
+# create a blink batch TableEnvironment
+table_env = BatchTableEnvironment.create(environment_settings=EnvironmentSettings.new_instance().in_batch_mode().use_blink_planner().build())
+
+# create a flink streaming TableEnvironment
+table_env = StreamTableEnvironment.create(environment_settings=EnvironmentSettings.new_instance().in_streaming_mode().use_old_planner().build())
+
+# create a flink batch TableEnvironment
+table_env = BatchTableEnvironment.create(environment_settings=EnvironmentSettings.new_instance().in_batch_mode().use_old_planner().build())
+
+{% endhighlight %}
+
+The `TableEnvironment` is responsible for:
+
+* Creating `Table`s
+* Registering `Table`s to the catalog
+* Executing SQL queries
+* Registering user-defined (scalar, table, or aggregation) functions
+* Offering further configuration options.
+* Add Python dependencies to support running Python UDF on remote cluster
+* Executing jobs.
+
+Currently there are 2 planners available: flink planner and blink planner.
+
+You should explicitly set which planner to use in the current program.
+We recommend using the blink planner as much as possible. 
+The blink planner is more powerful in functionality and performance, and the flink planner is reserved for compatibility.
```

Comment:
I'm not sure how soon the legacy planner will be removed, but if it is to be removed for 1.12, then I wonder if we even need to explain this here.

Reply:
I'll remove this line.